config/jaeger/deployment.yaml

```diff
@@ -0,0 +1,29 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: jaeger
```

Comment:
Just wondering if we need the Jaeger deployment , the docs seem to suggest its not supported , i.e. end users can install and configure along side of 3scale but we don't provide direct support for it https://access.redhat.com/documentation/en-us/red_hat_3scale_api_management/2.10/html/administering_the_api_gateway/introduction-to-advanced-operation-of-threescale-apicast-api-gateway_api-gateway-apicast#installing_jaeger_on_your_openshift_instance
having the deployment may confuse this point.

Reply:
This is only for the upstream operator. So that contributors can give it a try.

Reply:
guess we can add the warning just like the docs team to the upstream docs 
>**WARNING:** Jaeger is a third-party component, which 3scale does not provide support for, with the exception of uses with APIcast. The following instructions are provided as a reference example only, and are not suitable for production use.